debian/pantos-service-node.dirs

```diff
@@ -0,0 +1,2 @@
+var/log/pantos/service-node
+etc/pantos
```

Comment:
Missing new line at the end.

Reply:
We shouldn't add empty lines on this file